models/optional_modules/app_errors/snowplow_mobile_app_errors.sql

```diff
@@ -1,22 +1,22 @@
-{{ 
+{{
   config(
     materialized=var("snowplow__incremental_materialization"),
     unique_key='event_id',
     upsert_date_key='derived_tstamp',
     sort='derived_tstamp',
     dist='event_id',
-    partition_by = {
+    partition_by = snowplow_utils.get_partition_by(bigquery_partition_by={
       "field": "derived_tstamp",
       "data_type": "timestamp"
-    },
+    }, databricks_partition_by='derived_tstamp'),
```

Comment:
Actually here we should do the same as we do in the web package and just partition by derived_tstamp_date, as well as define it in the SQL as follows:
```SQL
  {% if target.type == 'databricks' -%}
  , DATE(derived_tstamp) as derived_tstamp_date
  {%- endif %}
```